e2e/tests/text/adding-text/index.spec.ts

```diff
@@ -0,0 +1,85 @@
+import { CanvasUtil } from '../../../utils/CanvasUtil';
+import { expect, test } from '@playwright/test';
+import { ObjectUtil } from '../../../utils/ObjectUtil';
+import type { Textbox } from 'fabric';
+import '../../../setup';
+
+[false, true].forEach((splitByGrapheme) => {
+  test(`adding new lines and copy paste - splitByGrapheme: ${splitByGrapheme}`, async ({
+    page,
+    context,
+  }) => {
+    await context.grantPermissions(['clipboard-read', 'clipboard-write']);
+    const canvasUtil = new CanvasUtil(page);
+    const textBoxutil = new ObjectUtil(page, 'text');
+
+    await textBoxutil.executeInBrowser(
+      (textbox: Textbox, context) => {
+        textbox.splitByGrapheme = context.splitByGrapheme;
+        textbox.set('dirty', true);
+        textbox.initDimensions();
+        textbox.canvas.renderAll();
+      },
+      { splitByGrapheme }
+    );
+
+    await expect(await canvasUtil.screenshot()).toMatchSnapshot({
+      name: `1-initial-splitByGrapheme-${splitByGrapheme}.png`,
```

Comment:
Please rename:
```suggestion
      name: `1-initial-${splitByGrapheme ? 'splitByGrapheme ': ''}.png`,
```

Reply:
also perhaps create a function to generate the name?

Reply:
i think snapshots name are fine as they are
<img width="399" alt="image" src="https://github.com/fabricjs/fabric.js/assets/1194048/54b82b70-500f-41cd-8d58-5278eb45da07">

They specify if splitByGrapheme is true or false explicitly,  without leaving to interpretation the one that doesn't say it what it is